config/motoros2_config.yaml

```diff
@@ -284,3 +284,35 @@ publisher_qos:
 #
 # DEFAULT: false
 #ignore_missing_calib_data: false
+
+#-----------------------------------------------------------------------------
+# Should MotoROS2 broadcast debug messages? 
+#
+# If enabled, this will broadcast log messages on the network on port UDP 21789.
+# the user can use the debug script provided in the distribution of MotoROS2 to 
+# monitor the state of the robot, identify problems, and debug their code.
+#
+# The debug script is available under the Yaskawa-Global/motoros2 repository in 
+# the tools directory
+# https://github.com/Yaskawa-Global/motoros2/tree/main/tools
+#
+# DEFAULT: true
+#userlan_debug_broadcast_enabled: true
+
+#-----------------------------------------------------------------------------
+# Which network port should MotoROS2 broadcast the debug messages to, if 
+# 'userlan_debug_broadcast_enabled' is 'true'?
+#
+# If not specified and 'userlan_debug_broadcast_enabled' is true, MotoROS2 will 
+# send the broadcast to all network ports which are enabled on the controller. 
+#
+# To choose a specific port to broadcast debug messages, uncomment 
+# 'userlan_debug_broadcast_port' below and set it to the desired port.
+#
+# NOTE 1:   this setting only applies to YRC1000 and YRC1000u controllers.
```

Comment:
YRC1000u only has one LAN port

Reply:
There is a lot of code outside of this PR assuming that the YRCu has 2 LAN ports. I'll fix it for this code, but there should probably be a second PR that fixes that up elsewhere.

Reply:
Tracking in #310.